easy-random-core/src/main/java/org/jeasy/random/api/RandomizerContext.java

```diff
@@ -70,4 +71,12 @@
      */
     EasyRandomParameters getParameters();
 
+    /**
+     * Get a random element from the list.
+     *
+     * @param list the input list
+     * @param <T>  the type of elements in the list
+     * @return a random element from the list or null if the list is empty
+     */
+    <T> T randomElementOf(List<T> list);
```

Comment:
In addition to being a breaking change, this does not belong to the contract of `RandomizerContext`. We can manage to have a random element from a list where this is used (ie in `FieldPopulator` and `ObjenesisObjectFactory`) without this method, see other comments.

Reply:
removed